docs/build/basics/fees.md

```diff
@@ -393,3 +393,15 @@ Final fees are determined by the surge factor on the network. The surge factor f
 **I use Blocto and I haven't paid any fees yet. Why is that?**
 
 That is because Blocto is acting as the payer for transactions. Self-custody wallets may have the user pay the transaction. Additionally, apps can sponsor the transaction if they choose.
+
+**Why would the same transaction have different fees when executed for different accounts?**
+
+Short explanation: Execution costs, among other things, also include the cost to read data from the storage and since the data stored vary from account to account, so does the execution costs and subsequently the transaction fees.
```

Comment:
```suggestion
Execution costs, among other things, include the cost to read data from account storage and since the data stored varies from account to account, so does the execution costs, and subsequently the transaction fees.
```